### PR TITLE
rqt_mav_manager and kr_crazyflie_interface migrated to ROS2 humble

### PR DESCRIPTION
**Pull request by pranavpshah**
Migrated the following two packages to ROS2 humble:
1. rqt_mav_manager
2. interface/kr_crazyflie_interface

Tried renaming the files using `git mv` but did not reflect in the commit.

### CODE REVIEW

**Comment by fcladera** [comment]
Hi! Please reword the commit messages to make them more descriptive and push force to the branch.

File: rqt_mav_manager/package.xml [hunk 1]
```diff
@@ -1,25 +1,32 @@
-<package format="2">
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>rqt_mav_manager</name>
-  <version>0.1.0</version>
+  <version>0.0.0</version>
```
Comment (fcladera) [inline rqt_mav_manager/package.xml]: Please update the version number to 0.2.0

File: rqt_mav_manager/resource/rqt_mav_manager
Comment (fcladera) [inline rqt_mav_manager/resource/rqt_mav_manager:1]: This file is empty, is this correct?

Comment (pranavpshah) [reply rqt_mav_manager/resource/rqt_mav_manager:1]: Yes, that's correct

File: rqt_mav_manager/package.xml (hunk 1 above)
Comment (pranavpshah) [reply rqt_mav_manager/package.xml]: Version updated and new commit pushed

**Review by fcladera — APPROVED** [review]

**Pull request merged**